conda/recipes/rmm/recipe.yaml

```diff
@@ -20,8 +20,23 @@ source:
 build:
   string: cuda${{ cuda_major }}_py${{ py_buildstring }}_${{ date_string }}_${{ head_rev }}
   script:
-    content:
-      - ./build.sh -v clean rmm --cmake-args=\"-DCMAKE_INSTALL_LIBDIR=lib\"
+    content: |
+      # Remove `-fdebug-prefix-map` line from CFLAGS and CXXFLAGS so the
+      # incrementing version number in the compile line doesn't break the
+      # cache
+      set -x
+      export CFLAGS=$(echo $CFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
+      export CXXFLAGS=$(echo $CXXFLAGS | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
+      set +x
```

Comment:
Have copied the stripping of the existing `-fdebug-prefix-map` over from `librmm`:

https://github.com/rapidsai/rmm/blob/f87d7a64e2451fc54a98bc267e556a371fcff1e3/conda/recipes/librmm/recipe.yaml#L21-L28

We would want this in both builds for the same reasons